workspaces/liferay-ticket-workspace/client-extensions/liferay-ticket-custom-element/src/main.tsx

```diff
@@ -3,26 +3,51 @@
  * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
  */
 
+import {ClayIconSpriteContext} from '@clayui/icon';
 import {createRoot} from 'react-dom/client';
 import {QueryClient, QueryClientProvider} from 'react-query';
 
-import TicketApp from './pages/TicketApp';
+import AllScreens from './pages/AllScreens';
+import TicketsDashboard from './pages/TicketsDashboard';
+import TicketsOverview from './pages/TicketsOverview';
+import {Liferay} from './services/liferay';
+import {ScreenType} from './types';
 
-const queryClient = new QueryClient();
-class TicketWebComponent extends HTMLElement {
+export type LiferayTicketWorkspaceComponentsType = {
+	[key: string]: JSX.Element;
+};
+
+const NoRouteSelected = () => <AllScreens />;
```

Comment:
Why do we need to create an extra component for this. Can't we just directly use AllScreens instead.